backend/pkg/logger/data/logger.go

```diff
@@ -0,0 +1,115 @@
+package data
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/logger"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/transport/packet/data"
+)
+
+type ValueName string
+
+const (
+	name abstraction.LoggerName = "data"
+)
+
+type Logger struct {
+	running    bool
+	lock       sync.Mutex
+	valueFiles map[data.ValueName]*os.File
```

Comment:
```suggestion
	outputs map[data.ValueName]*io.WriteCloser
```
With the `io.WriteCloser` interface you get all the functionality you want from the file and the added benefit of being able to put something other than a file